db/video/cfr/README.txt

```diff
@@ -0,0 +1,11 @@
+Create video: 
+ffmpeg -f lavfi -i color=c=blue:s=1280x720:d=3:r=60 -c:v libx264 -vf "format=pix_fmts=yuv420p, drawtext=fontsize=480: fontcolor=white: font=monospace: x=(w-text_w)/2: y=(h-text_h)/2: r=60: text='%{frame_num}'" test.mp4
+
+Get frames from video:
+ffmpeg -i test.mp4 %03d.png
+
+Build video from frames:
+ffmpeg -r 60 -i %03d.png -c:v libx264 -g 10 -keyint_min 10 -vf fps=60 -pix_fmt yuv420p test.mp4
+
+Transcode file to VFR, add keyframes:
+ffmpeg -i test_1.mp4 -vsync vfr -vf setpts='N/(25*TB)' -g 10 -keyint_min 10  test_2.mp4
```

Comment:
The cfr readme mentions vfr video. Is this intended?
Also here is the only mention of test_2, while we have test_1 and test_2.
Can you maybe add info how the cfr/test_1 and test_2 correspond to vfr/test_1 and test_2?

Reply:
Done. Added second README to vfr, made command general (input.mp4, output.mp4 as file names) and explained how it was used to obtain the files.